### ISSUE TITLE
Give better documentation about dealing with existing bazelrc file.

### ISSUE THREAD

**Issue by dmaclach** [google/bazel#196]
Right now it says 

"You already have a .bazelrc. please modify it to add  /Users/foo/src/bazel_ext/bazel/base_workspace to your build package path."

It would be nice to actually give more details. What info do I actually have to add to the file? Offer to do it for me? These steps should be made as easy as possible for people coming to bazel.

**Comment by damienmg** [comment]
This message was just modified I believe. It should says exactly the lines you need to have in your .bazelrc now.

**Comment by kchodorow** [comment]
For reference, it should be:

```
build --package_path %workspace%:/path/to/bazel/base_workspace
fetch --package_path %workspace%:/path/to/bazel/base_workspace
query --package_path %workspace%:/path/to/bazel/base_workspace
```

(Replace `/path/to/bazel` with the path to wherever you downloaded Bazel.)

**Comment by kchodorow** [comment]
Fixed by 45a24691406c683cfab5bed30f25de8ef2a8300b.
